src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+}
+
+.header__link,
+.nav__list {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.nav__list {
+  justify-content: center;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  position: relative;
+  font-size: 12px;
+  text-transform: uppercase ;
+  margin-left: 20px;
+  transition: color 0.3s;
+  font-weight: 500;
+  font-family: Roboto , Helvetica , sans-serif;
+  font-style: normal;
+  color: #060b35;
+
+  &:hover {
+    color: #00acdc;
+    cursor: pointer;
+  }
+}
+
+.nav__item:first-of-type {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 60px;
+  color: inherit;
+  text-decoration: none;
+  line-height: 14px;
+
+}
+
+a.is-active {
+  color: #00acdc;
+}
+
+a.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+  background-color: #00acdc;
+}
```

Comment:
please leave `.is-active` class without a - we might want to animate not only `<a>` tags